### ISSUE TITLE
Is it ok that ScriptHandler::installAssets uses hardcoded folders?

### ISSUE THREAD

**Issue by Boorj** [bolt/bolt#4406]
Script Handler uses, for example, cache folder as hardcoded to 'app/cache': 
https://github.com/bolt/bolt/blob/master/src/Composer/ScriptHandler.php#L74
```php
$filesystem->mkdir($appDir . '/cache/',    $dirMode);
```
And in https://github.com/bolt/bolt/blob/master/src/Configuration/ResourceManager.php#L92 it's being set by path, so it could be changed on init.
```php
$this->setPath('cache', 'app/cache');
```
I put logs and cache to `app/vars/cache`. So the question is : am i missing something in docs? Can i  change cache folder? If so - ok. Probably i could even write my own script handler (i saw that advice [here](https://docs.bolt.cm/installation-advanced))

**Comment by SahAssar** [comment]
@Boorj i'm sorry, but since you have been using dev versions of bolt for quite a while I'm going to ask you to reverify some of the bugs you filed, this being one of them...

Could you check that this is still an issue and in that case update this issue?

**Comment by GawainLynch** [comment]
Fixed in #5069
